src/main/java/lotto/domain/LottoNumberFactory.java

```diff
@@ -0,0 +1,26 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoNumberFactory {
+	private static final int FIRST_INDEX = 0;
+	private static final int SIX_INDEX = 6;
+	private static final int MIN_LOTTO_NUMBER = 1;
+	private static final int MAX_LOTTO_NUMBER = 45;
+	private static final List<Integer> allLottoNumbers = new ArrayList<>();
+
+	static {
+		for (int i = MIN_LOTTO_NUMBER; i <= MAX_LOTTO_NUMBER; i++) {
+			allLottoNumbers.add(i);
+		}
+	}
+
+	public static List<Integer> create() {
```

Comment:
```suggestion
	public static Lotto create() {
```

로또를 생성하는 기능을 가졌으니 Lotto 클래스의 일급컬렉션을 만들어 Lottos를 반환해보는건 어떨까요?
클래스 이름도 수정이 필요할 수도 있을 것 같네요 :)